src/NzbDrone.Core/MediaFiles/MediaInfo/VideoFileInfoReader.cs

```diff
@@ -18,8 +18,8 @@ public class VideoFileInfoReader : IVideoFileInfoReader
         private readonly IDiskProvider _diskProvider;
         private readonly Logger _logger;
 
-        public const int MINIMUM_MEDIA_INFO_SCHEMA_REVISION = 3;
-        public const int CURRENT_MEDIA_INFO_SCHEMA_REVISION = 5;
+        public const int MINIMUM_MEDIA_INFO_SCHEMA_REVISION = 4;
```

Comment:
Keep the minimum for Sonarr at 3, Radarr minimum was bumped on v0.2 in a previous change